src/App.js

```diff
@@ -1,27 +1,27 @@
-import React, { useState } from "react";
+import { useState } from "react";
 import FeatherIcon from "feather-icons-react/build/FeatherIcon";
 import { useDispatch, useSelector } from "react-redux";
-import ListTodos from "./components/ListTodos";
-import { add, clear, remove } from "./features/toDoList";
+import ListTodos from "./components/Todo";
+import { add, clear, remove } from "./features/todos";
 
-export default function Todos() {
-  const toDoList = useSelector((state) => state.toDolist);
+export default function App() {
+  const toDoList = useSelector((state) => state.todosList);
   const dispatch = useDispatch();
-  const [inputValue, setInputValue] = useState();
+  const [inputValue, setInputValue] = useState("");
```

Comment:
A one-line gap between this line and the return would make the code much easier to read.